src/test/java/com/spring/boot/project/demo/service/impl/ReviewServiceImplTest.java

```diff
@@ -22,13 +25,11 @@
 import org.mockito.Mockito;
 import org.mockito.junit.MockitoJUnitRunner;
 import org.springframework.boot.test.context.SpringBootTest;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
 
 @RunWith(MockitoJUnitRunner.class)
 @SpringBootTest
 public class ReviewServiceImplTest {
-    private final List<String> EXPECTED_MOST_USED_WORDS = List.of("review", "text");
+    private static final List<String> EXPECTED_MOST_USED_WORDS = List.of("review", "text");
```

Comment:
it's not a constant, let's change naming 
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
Am I need to change name or change List to ImmutableList? Are we need use this field as constant or can we use this just as variable?